lib/tasks/panopticon.rake

```diff
@@ -85,4 +85,22 @@ namespace :panopticon do
       end
     end
   end
+
+  desc "Re-register all content with Panopticon"
+  task :re_register_all_content => :environment do
+    logger = GdsApi::Base.logger = Logger.new(STDERR).tap { |l| l.level = Logger::INFO }
+
+    published_editions = Edition.all
+
+    published_editions.each do |edition|
```

Comment:
Using `.find_each` here will prevent rails from loading everything into memory, which can be slow for large data sets.